corecito.py

```diff
@@ -54,19 +62,37 @@ async def main():
 
       elif coreNumberIncreased(account.core_number, deviated_core_number, account.min_core_number_increase_percentage, account.max_core_number_increase_percentage):
         logger.logCoreNumberIncreased(increase_percentage, excess, account.core_number_currency, account.base_currency, telegram)
-        tx_result = round(excess * buy_price, account.max_decimals_buy)
+        #Check if 'fiat' is True to adjust messages format and tx_result var and 'excess' has to be divided by the buy_price
+        if fiat:
+          tx_result = round(excess / buy_price, account.max_decimals_buy)
+        else:
+          tx_result = round(excess * buy_price, account.max_decimals_buy)
         logger.logSellExcess(tx_result, account.base_currency, buy_price, excess, account.core_number_currency, telegram)
+
+  
         # Sell excess of base currency ie. => in ETH_BTC pair, sell excess BTC => Buy ETH
+        # If fiat, we sell the value previously calculated and stored on tx_result
         if (not config['safe_mode_on']):
-          await account.order_market_buy(tx_result, excess)
+          if fiat:
+            await account.order_market_sell(tx_result)
+          else:
+            await account.order_market_buy(tx_result, excess)
 
       elif coreNumberDecreased(account.core_number, deviated_core_number, account.min_core_number_decrease_percentage, account.max_core_number_decrease_percentage):
-        logger.logCoreNumberDecreased(decrease_percentage, missing, account.core_number_currency, account.base_currency, telegram)
-        tx_result = missing * sell_price
-        logger.logBuyMissing(tx_result, account.base_currency, buy_price, missing, account.core_number_currency, telegram)
```

Comment:
It seems the merge deleted the 2 logger lines that should replace the old logger.info lines below.

Reply:
ummm, I'm on it!